site/public/js/server.js

```diff
@@ -169,11 +223,19 @@ function loadTestcaseOutput(div_name, gradeable_id, who_id, index, version = '')
                 enableKeyToClick();
             },
             error: function(e) {
-                alert("Could not load diff, please refresh the page and try again.");
-                console.log(e);
-                displayAjaxError(e);
+                if(output_xhrs.has(orig_div_name) && output_xhrs.get(orig_div_name).readyState == 0){
```

Comment:
Same comment about strict equality operator here
```suggestion
                if(output_xhrs.has(orig_div_name) && output_xhrs.get(orig_div_name).readyState === 0){
```